src/test/java/ruslan/shastkiv/bookstore/utils/OrderTestUtils.java

```diff
@@ -0,0 +1,248 @@
+     *               {@code null}.
+     * @return an {@link OrderItemDto} with the fields initialized based on the provided
+     *         {@code bookId}.
+     */
+
+    public static OrderItemDto createOrderItemDto(Long bookId) {
+        return new OrderItemDto(
+                bookId,
+                bookId,
+                bookId.intValue()
+        );
+    }
+
+    /**
+     * Creates an {@link OrderDto} for the specified user and populates it with
+     * {@link OrderItemDto}s based on the provided list of item IDs.
+     * The {@link OrderDto} ID will be the same as the user's ID
+     * for simplicity. The total cost is calculated as the sum of the
+     * products of the item quantities and their IDs
+     * (used as a placeholder for price in this test utility).
+     *
+     * <p>The created {@link OrderDto} will include:
+     * - The same ID for both the order and the user.
+     * - A list of {@link OrderItemDto}s created based on the provided item IDs.
+     * - A default order date.
+     * - The calculated total cost based on the item quantities and IDs.
+     * - The specified order status.
+     *
+     * @param userId  the ID of the user who placed the order;
+     * @param itemIds a list of item IDs to be included in the order;
+     * @param status  the status of the order.
+     * @return an {@link OrderDto} instance with the specified user ID, items, and status.
+     */
+
+    public static OrderDto createOrderDtoWithItems(Long userId,
+                                                   List<Long> itemIds,
+                                                   Order.Status status) {
+        List<OrderItemDto> itemDtos = itemIds.stream()
+                .map(OrderTestUtils::createOrderItemDto)
+                .toList();
+        return new OrderDto(userId,
+                userId,
+                itemDtos,
+                ORDER_DATE,
+                BigDecimal.valueOf(itemDtos.stream()
+                        .mapToDouble((orderItem -> orderItem.id() * orderItem.id()))
+                        .sum()),
+                status);
+    }
+
+    /**
+     * Creates an {@link Order} for the specified {@link User} and populates it with
+     * {@link OrderItem}s based on the provided list of book IDs.
+     * The order is initialized with default values for status,
+     * order date, and shipping address. The total cost is calculated
+     * as the sum of the prices of all items multiplied by their quantities.
+     *
+     * <p>The created {@link Order} will have the same ID as the provided user.
+     * The created {@link OrderItem} will have the same ID as the associated book.
+     *
+     * @param user    the {@link User} for whom the order is created;
+     * @param bookIds a list of book IDs to be added to the order as items;
+     * @return an {@link Order} instance with the specified user and populated items.
+     */
+
+    public static Order createOrderWithItems(User user, List<Long> bookIds) {
```

Comment:
This should be a simple method that simply returns you an instance of the `Order` model with only the data you need. Don't complicate things